scripts/linux/nv/gitlab_test_performance.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/bash -ex
```

Comment:
Just asking. Isn't `#!/bin/env bash` a better version of shebang?

Reply:
there is one drawback, it’s a bit hard to track -x and -e in case we would use new way.